modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/SCIM2RestClient.java

```diff
@@ -85,6 +86,31 @@ public String createUser(UserObject userInfo) throws Exception {
         }
     }
 
+    /**
+     * Create a user inside an organization.
+     *
+     * @param userInfo         Object with user creation details.
+     * @param switchedM2MToken Switched M2M token for the given organization.
+     * @return ID of the created user.
+     * @throws Exception If an error occurred while creating the user.
+     */
+    public String createSubOrgUser(UserObject userInfo, String switchedM2MToken) throws Exception {
+
+        String jsonRequest = toJSONString(userInfo);
+        if (userInfo.getScimSchemaExtensionEnterprise() != null) {
+            jsonRequest = jsonRequest.replace("scimSchemaExtensionEnterprise",
```

Comment:
Shall we add contants to these places

Reply:
Addressed with https://github.com/wso2/product-is/pull/21184/commits/6b46b01e32048142b492dd92ffffa01822d5eab8